docs/sources/enterprise/reporting.md

```diff
@@ -136,12 +138,33 @@ image_scale_factor = 2
 
 ## Report time range
 
-Reports use the saved time range of the dashboard. Changing the time range of the report is done by saving a modified time range to the dashboard.
+> Setting custom report time range is available in Grafana Enterprise v7.2+.
+
+By default, reports use the saved time range of the dashboard. Changing the time range of the report can be done by:
+- Saving a modified time range to the dashboard. 
+- Setting a time range via **Time range** field in the report form. If specified, this custom time range will override the one from the report's dashboard.
 
 The page header of the report displays the time range for the dashboard's data queries. Dashboards set to use the browser's time zone will use the time zone on the Grafana server.
 
 If the time zone is set differently between your Grafana server and its remote image renderer, then the time ranges in the report might be different between the page header and the time axes in the panels. We advise always setting the time zone to UTC for dashboards when using a remote renderer to avoid this. 
 
+## Reports settings
+
+> Only available in Grafana Enterprise v7.2+.
+
+Organisation-wide report settings can be configured via **Settings** tab on the **Reporting** page. They will be applied to all the reports for current organisation.
+
+Currently customisation is available for the branding options.  
+
+Report branding:
+1. **Company logo URL** - Company logo, displayed in the report PDF. Defaults to Grafana logo.
+
+Email branding:
+1. **Company logo URL** - Company logo, displayed in the report PDF. Defaults to Grafana logo.
+1. **Email footer** - Text for the report email footer: **Sent by** or **None**.
```

Comment:
```suggestion
1. **Email footer** - Text for the report email footer. Select **Sent by** or **None**.
```

Reply:
I assume you can enter something if you select **Sent by**. Is there a character limit here? Is HTML or markdown allowed?

Reply:
This entire list should be a bulleted list, not a numbered list.

Use a numbered list if the items are sequential and the order matters, as in a procedure. Otherwise, use an unordered list.

Reply:
> I assume you can enter something if you select **Sent by**. Is there a character limit here? Is HTML or markdown allowed?

It's just a text field where one would normally enter their company's name. I'll add some screenshots, it seems that has to be in the website repo, so they will hopefully make it more clear.